haystack/nodes/retriever/base.py

```diff
@@ -155,7 +159,13 @@ def eval(
 
         timed_retrieve = self.timing(self.retrieve, "retrieve_time")
 
-        labels: List[MultiLabel] = self.document_store.get_all_labels_aggregated(
+        if document_store is None:
```

Comment:
Why not use a shortcut here? `document_store = document_store or self.document_store` then if `document_store` is None raise error